dGame/dBehaviors/DarkInspirationBehavior.cpp

```diff
@@ -0,0 +1,37 @@
+#include "DarkInspirationBehavior.h"
+
+#include "BehaviorBranchContext.h"
+#include "Entity.h"
+#include "DestroyableComponent.h"
+#include "EntityManager.h"
+#include "BehaviorContext.h"
+
+void DarkInspirationBehavior::Handle(BehaviorContext* context, RakNet::BitStream* bitStream, const BehaviorBranchContext branch) {
+	auto* target = EntityManager::Instance()->GetEntity(branch.target);
+
+	if (target == nullptr) {
+		Game::logger->Log("DamageReductionBehavior", "Failed to find target (%llu)!", branch.target);
```

Comment:
Wrong log file name, also, maybe make debug?

Reply:
ah right, just copy and paste error